code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -245,6 +245,37 @@
 			L.extract_cooldown = max(0, L.extract_cooldown - 20)
 	..()
 
+/datum/reagent/teslium/energized_jelly/energized_ooze
+	name = "Energized Ooze"
+	description = "Electrically-charged Ooze. Boosts Oozeling's nervous system, but only shocks other lifeforms."
+	reagent_state = LIQUID
+	color = "#CAFF43"
+	taste_description = "slime"
+	overdose_threshold = 30
+
+/datum/reagent/teslium/energized_jelly/energized_ooze/on_mob_life(mob/living/carbon/M)
+	if(isjellyperson(M))
+		shock_timer = 0 //immune to shocks
+		M.AdjustAllImmobility(-40, FALSE)
+		M.adjustStaminaLoss(-2, 0)
+		if(isluminescent(M))
+			var/mob/living/carbon/human/H = M
+			var/datum/species/jelly/luminescent/L = H.dna.species
+			L.extract_cooldown = max(0, L.extract_cooldown - 20)
```

Comment:
```suggestion
```

The exact same code is at `/datum/reagent/teslium/energized_jelly/on_mob_life`. Excepting Jellypeople will be handled there; you called `..()`